### PR TITLE
add-links-to-other-pages-to-qts

### PR DESCRIPTION
**Pull request by joegibb**
### add text link to 
train to teach if you have a degree 
get school experience page 
teaching internships

### CODE REVIEW

File: app/views/content/train-to-be-a-teacher/teaching-internship-providers/_listing.html.erb [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 <p>Find out how to apply, including details of teaching internship providers in each region, further down this page.</p>
 
+<p>Once you have finished your internship, you can decide if you want to train to teach in England and gain your<a href="/train-to-be-a-teacher/what-is-qts">qualified teacher status (QTS)</a>.</p>
```
Comment (Sarah-DfE) [inline app/views/content/train-to-be-a-teacher/teaching-internship-providers/_listing.html.erb:7]:
```suggestion
<p>Once you've finished your internship, you can decide if you want to train to teach in England and gain your <a href="/train-to-be-a-teacher/what-is-qts">qualified teacher status (QTS)</a>.</p>
```

**Review by Sarah-DfE — COMMENTED** [review]
Looks good, made a few mini amends

**Comment by Sarah-DfE** [comment]
For the experience page, could you update this link if poss, so it tells users where it's going? 
image

File: app/views/content/train-to-be-a-teacher/get-school-experience.md [hunk 2]
```diff
@@ -99,3 +99,7 @@ There are other ways to gain valuable experience of working with young people. S
 For example, you could get paid to work in a school that offers childcare before and after school. This is known as wraparound because the childcare wraps around the school day.
 
 Working in wraparound gives you experience working with children and in a school environment. Contact your local council or school to find out about any available opportunities. 
+
+## Next steps 
+
+Once you have completed your school experience, you can decide if you want to apply to teacher training in England and gain your qualified teacher status (QTS). [Find out more about the benefits of gaining QTS](/train-to-be-a-teacher/what-is-qts).
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/get-school-experience.md:105]:
```suggestion
Once you have completed your school experience, you can decide if you want to apply to teacher training in England and get qualified teacher status (QTS). [Find out more about the benefits of gaining QTS](/train-to-be-a-teacher/what-is-qts).
```
we don't usually talk about `**your** QTS` :-)

File: app/views/content/train-to-be-a-teacher/teaching-internship-providers/_listing.html.erb (hunk 1 above)
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/teaching-internship-providers/_listing.html.erb:7]:
```suggestion
<p>Once you have finished your internship, you can decide if you want to train to teach in England and get <a href="/train-to-be-a-teacher/what-is-qts">qualified teacher status (QTS)</a>.</p>
```

**Review by gemmadallmandfe — COMMENTED** [review]
Looks good @joegibb - couple of very minor comments!

File: app/views/content/train-to-be-a-teacher/get-school-experience.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/train-to-be-a-teacher/get-school-experience.md:105]: Can we say `get QTS` and `getting QTS` though?

Also - in Sarah's suggested change, the space between the link text and link will need removing :-)

**Review by Sarah-DfE — APPROVED** [review]
Looks good to me @joegibb

**Pull request merged**